preprocessing.py

```diff
@@ -106,6 +104,7 @@ def batch_parse_tf_example(batch_size, example_batch):
 
 def read_tf_records(batch_size, tf_records, num_repeats=None,
                     shuffle_records=True, shuffle_examples=True,
+                    shuffle_buffer_size=10000,
```

Comment:
another instance of that constant

Reply:
I don't want this to depend on dual_net, but i don't want to not have a default value.